bundles/org.openhab.binding.miio/src/main/resources/database/ijai.vacuum.v19-miot.json

```diff
@@ -0,0 +1,2543 @@
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"hypa-hours",
+				"siid":7,
+				"piid":13,
+				"friendlyName":"Sweep - Hypa Hours",
+				"channel":"hypa_hours",
+				"type":"Number:Time",
+				"unit":"hours",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":180,
+					"step":1,
+					"pattern":"%.0f %unit%",
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"mop-life",
+				"siid":7,
+				"piid":14,
+				"friendlyName":"Sweep - Mop Life",
+				"channel":"mop_life",
+				"type":"Number:Dimensionless",
+				"unit":"percentage",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":100,
+					"step":1,
+					"pattern":"%.0f %unit%",
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"mop-hours",
+				"siid":7,
+				"piid":15,
+				"friendlyName":"Sweep - Mop Hours",
+				"channel":"mop_hours",
+				"type":"Number:Time",
+				"unit":"hours",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":180,
+					"step":1,
+					"pattern":"%.0f %unit%",
+					"readOnly":true
+				},
+				"refresh":true,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"direction",
+				"siid":7,
+				"piid":16,
+				"friendlyName":"Sweep - Direction",
+				"channel":"direction",
+				"type":"Number",
+				"stateDescription":{
+					"options":[
+						{
+							"value":"1",
+							"label":"Forward"
+						},
+						{
+							"value":"2",
+							"label":"Left"
+						},
+						{
+							"value":"3",
+							"label":"Right"
+						},
+						{
+							"value":"4",
+							"label":"Back"
+						},
+						{
+							"value":"5",
+							"label":"Stop"
+						},
+						{
+							"value":"10",
+							"label":"Exit"
+						}
+					]
+				},
+				"refresh":false,
+				"actions":[
+					{
+						"command":"set_properties",
+						"parameterType":"NUMBER"
+					}
+				],
+				"readmeComment":"Value mapping [\"1\"\u003d\"Forward\",\"2\"\u003d\"Left\",\"3\"\u003d\"Right\",\"4\"\u003d\"Back\",\"5\"\u003d\"Stop\",\"10\"\u003d\"Exit\"]"
+			},
+			{
+				"property":"time-zone",
+				"siid":7,
+				"piid":20,
+				"friendlyName":"Sweep - Time Zone",
+				"channel":"time_zone",
+				"type":"Number",
+				"stateDescription":{
+					"step":1,
+					"pattern":"%.0f"
+				},
+				"refresh":true,
+				"actions":[
+					{
+						"command":"set_properties",
+						"parameterType":"NUMBER"
+					}
+				]
+			},
+			{
+				"property":"cur-lang",
+				"siid":7,
+				"piid":21,
+				"friendlyName":"Sweep - Cur Lang",
+				"channel":"cur_lang",
+				"type":"String",
+				"refresh":true,
+				"actions":[
+					{
+						"command":"set_properties",
+						"parameterType":"STRING"
+					}
+				]
+			},
+			{
+				"property":"cleaning-time",
+				"siid":7,
+				"piid":22,
+				"friendlyName":"Sweep - Cleaning Time",
+				"channel":"cleaning_time",
+				"type":"Number",

```

Comment:
Is this like the time spend cleaning. In that case you may want to change the type to some Number:Time and add a unit